src/tools/rest-frontend/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ else ()
     # install files to destination
     install (DIRECTORY public resources DESTINATION ${install_directory})
     install (FILES package.json README.md ${CMAKE_CURRENT_BINARY_DIR}/Gruntfile.js DESTINATION ${install_directory})
-    if (NOT EXISTS ${install_directory}/application-config.json)
+    if (NOT EXISTS ${CMAKE_INSTALL_PREFIX}/${install_directory}/application-config.json)
```

Comment:
I am afraid this will not work as you expect it: The test is already done at cmake time, which is too early. (You want it at installation time)

Two config files would be a much better way.